securedrop/requirements/test-requirements.txt

```diff
@@ -27,9 +22,7 @@ pip-tools==1.9.0
 pluggy==0.6.0             # via pytest
 py==1.5.2
 pytest-cov==2.5.1
-pytest==3.3.1
```

Comment:
Regression from 3.3.1 to 3.1.1 is probably not intended ? It will cause trouble with the caplog API for tests

Reply:
Ahh.. good call. Fixing

Reply:
Updated :)